.scrutinizer.yml

```diff
@@ -0,0 +1,23 @@
+tools:
+    php_sim: false
+    php_pdepend: true
+    php_analyzer: true
+
+build:
+    environment:
+        php:
+            version: 5.6.0
+    tests:
+        override:
+            -
+                command: 'vendor/bin/phpunit --verbose --configuration phpunit.xml --coverage-clover test/clover.xml'
```

Comment:
I never used Scrutinizer before, but I think this is going to execute again the unit tests after Travis. Isn't there a way to upload the `clover.xml` file from Travis to Scrutinizer? This way we would run the test suite only once